src/components/select/select.tsx

```diff
@@ -23,23 +23,35 @@ type Props = {
     onChange?: (value: string) => void
     /** Options that are rendered in the select. */
     options?: Option[]
+    /** Classname for the option elements */
+    optionsClassName?: string
 }
 
-function Select({ value, options = [], onChange, disabled = true, className = '' }: Props) {
+function Select({
+    value,
+    options = [],
+    onChange,
+    disabled = true,
+    className = '',
+    optionsClassName,
```

Comment:
I was going to object to adding this prop in the time picker, but I did not do so because I saw it being passed to the Select, and thought it was already available there.

But now I see it is being added here too.

My objection is not super strong, because I still think all these components have a lack of composability in how their APIs are designed. But I still think it may not be needed even in the short term. You can always apply a class name to the parent select, and in css you can style the options by using a selector such as `.selectClassName option`.

Reply:
> and in css you can style the options by using a selector such as .selectClassName option

I was already doing this when the component was in twist, but once I moved it to reactist, the css I had for this stopped working and the only way I could get it styled was to do it like this 🤷🏻‍♂️

Reply:
Ok, not sure what was the problem in that case, but we should not add a prop to a public API in a library for a short term fix. Once you add it, it becomes a breaking change to take it back. We need to figure out why it does not work.